ospool.osg-htc.org/development/frontend-template.xml

```diff
@@ -142,6 +142,10 @@
             <file absfname="/opt/osg-flock/ospool-pilot/itb/pilot/advertise-userenv" type="run:singularity" after_entry="True" const="True" executable="True" prefix="NOPREFIX" untar="False" wrapper="False">
                <untar_options cond_attr="TRUE"/>
             </file>
+            <file absfname="/opt/osg-flock/ospool-pilot/itb/pilot/hello-world" after_entry="True" const="True" executable="True" period="0" untar="False" wrapper="False">
```

Comment:
```suggestion
            <file absfname="/opt/osg-flock/ospool-pilot/itb/pilot/hello-world" after_entry="True" const="True" executable="True" period="0" prefix="NOPREFIX" untar="False" wrapper="False">
```

glideinWMS arcana... `prefix="NOPREFIX"` is required to have no prefix 😂

Reply:
it says in the [custom scripts page](https://glideinwms.fnal.gov/doc.prd/factory/custom_scripts.html) that "By default the prefix is GLIDEIN_PS_ but you can change that by setting "prefix" in the file section of the Frontend or Factory configuration."

Reply:
That's probably ok. Then when we look at the glideins' machine ads in the collector, we'll be looking for `GLIDEIN_PS_helloworld == true`.

Reply:
Oh I see, I'll keep it as NOPREFIX